crowd.go

```diff
@@ -121,11 +121,11 @@ func (api *API) GetUser(userName string, withAttributes bool) (*User, error) {
 // Login attempts to authenticate a user with the given username and password.
 // It constructs a URL with the given username and sends a POST request to the usermanagement authentication API with the provided password.
 // It returns a pointer to a User object with the user's information on successful authentication, or an error if authentication failed or an unknown error occurred.
-func (api *API) Login(userName, password string) (*User, error) {
+func (api *API) Login(userName, passWord string) (*User, error) {
```

Comment:
This change is unrelated to the issue, please do not change the naming of function arguments.

Reply:
Sure, but the argument `password` is same with `type password struct`, and it should be same form with `userName`.

Reply:
Okay, thanks.